pupil-spa/src/app/submission-pending/submission-pending.component.html

```diff
@@ -1,18 +1,17 @@
N3dNg/dYvTQpjTdZlbdYJGw5pLbTLwNZtHQpw/dcwLQpz3cnLYNcRi9dZLf8NfA3Ibg3Yji3Ygx3JzGDX4pDU07DYjO3Xjr3ZNhAKkS3Zk23W4IDT1IDZmf0JnJ3aoPDZoB3aNT3aGl0Npu3IqJ3aqu0JrN3arr3QsG3P1jDbtI0Jtm3bn5DbdmwNF93b0owNwB3HoDDcxI3bxo3GyM3b3aDO3dDczu0J0B3dmDDd1H0N/+wN3uwN2k3Gtd3dnC3d4K3F3XDO5K3L33De6C3c6n3b393e7v3ewxzflgwO9D3G6X3fm83e+r3E39Df163I4hDgVjzgBA7YBn7gPgwOq+zfYNzgDo7EEB7hf13cFI7g4TDJGF7D6LDhHM7dHl7gEx7iW5zgeVziE5z/Diie4va94o/d4i5+x+OwxjJ+wDRe4zr83Die4/m94y+cDk/8DRm8DkI+5B1e5F+92kj+4j0ewkwO5E7+5Dis2VIO11Re5S28Dhas3OnrDlyuwl7+5W/t2WI+5mROwGbevWie5mVM5Gze5mH+5oXcDv6L2HQ+D3Y+x2ue51AN2Xze5+wgv5V9vfUw6ISO54Z+6IjO5/Rgvo1evPYA6YFc6JMe1m6e6DA8D9qL1rl7D5y+w6Lw6VM+CqKu6O7gvOhw6qjO6V7N6k8t169+D8KbDqfrD7Z+67gO6pVu6byuujf9uP+Q6qUw7EAt1rveD55b0vb76JBuCs5O7MX+5v8w/7++LsDAHuzCnu0tLdWiDhDti9QUDBDMju3kHtjQfu7onrkZbbYCIe7N/u7lbu7yPu9QTO1vfO/4Pu7vjgqvPuoBwcb1fMkHMfAET+6pcPAHEbXpLLIJ0e5Ore+dbfDRPvHEDBDf7BAO//DZLtQdjxAW+xADDREYn/H6vgonnxAAqxHtahEt7/IFb/L9DhHnKjMjn+8azwoSf6o3j/MQD/MxP6o/D/QvL/RJv49Fb/Ql7/Q7H6pRL/XO7gpDL6pX7+5Br/VVn49d7/VNH9NPb49Lj/XDXgtbj49jT/Y5D/ZhX49pr/a4bgttT/d1D/dHz/ZzH49vb/esfgt5L4+Bz//3U4/3f++Oh4/4WU/4Z9+Oe48KGh/UuFD4kj/5gv/puYD57Kj5mz/puuD56dj4oW/ouxD552j6p5/nqa/65cj6jr/2r5/ojA/6lF/5vED65Ij7uf/1tW/76ij7rc/mvcD74ej7v1/2wW/sq6/8xf/lvwD73Uj8yx/3u0/92wj919/3x6/92Gj93Z/437/43Mj94//40w/+1Sj+6U/7xMD+y+j+73/3xYD81kj/0S/lxwAIP4KDhIWGh4iJioVBjY6PkJGSk5SVlpeYmZqbnJ2en6ChoqOWO6anqKmqq6ytrqmksZE2tLW2t7i5uru3sr6Ni8HCw4u/xsfIycrLzM3/mK/Q0dKtzp+819jZttWexN7fwtzi4+Tl5ueS0+rr0OiW2vDxve6U4Pb3hPT6+/z9/ewAA5ryB0meQXgEH+FbCC6hw4cQI24SSHEdxIMYr0VkyDGcxI8gQ/qrSDJaxIwoc23syDKRyJcwYzorSZPaxZQ4aUlsyZORzJ9Ag36qSRTWyZw4d/Zc+kOo06dQHxWd+hFpUqVMeUbdyvXlVKoSrab8mLVn17NoE34tClIsSpBltaadS9fcWqJt3WIMGZdl3b+Amd2tGVLvXrh9OwZezDjWYJqFDRsUmVhx48uYMz0uKVLyZL6VGWYeTTrS5oovPct7GVp06deYT6PurBoh/+vW92DrXiybIszatinjzr27+NzeAn8DzxZzOHHj0KMiDxhzOXOYzu1F3/50OjuZ1rHJzN6Qu/mY3r9XD89rPHlv5+OLTK/uJ/v27t8Pk88fIv364N2nC1D6EdPfgf/8Jw1QAg74U4H7ISihOwouaF+DuAQFoUcTdihOhSYxiOE8D25YjIcozgSiK0KNSGKJJrqU4ozIrPhKiy7WIlSMitDo4y822hRUjjruyCMiPyYpSpBCikjkU0ceouSUnjC5ylNE6uRUlIZQ6aUmVqqCZZZQcenTl2imEyYqUGVpQ5lmDpLmnJCseUpUbkYVpyB09hmEnQO1SSace/o5J6Bb5f+pZ6GGfgnoDokOCpUMjDZKJaKRPrlVpZYqaWdXim5qZqeXrgmqpItySeqUpp6qKVejrppkq1yFKmqUss5qJVq23spjrj+GySuqsB4JrI+7DvtqV7geOyOTc/VabIzO0hhktMROa2K1z65Il7TaFshttyB+my2zG46borfmLovWtuqiWG67Of6VbrweVvgXuOjqh++69AHGb7/k/StvwPue+27BBh88XWADEzxcww73tljEEodGMcCyMYZxxn1tTO5mHiuc1sQiczxYYx+DnFXK1q7MsslzVQYzsmth1vJZId+MM16Z7cwzUz6zCtloQi/cUtGl+lZa0idzxHSaAOnJBnXNz0196I3GXV0XfFqHTa+L3CEp9tljj4j22myL4nXbcKP9dtx0Tz133XjDfHfefDe8d9+Aq/t34IQfO3jhiJN6eOKM+7l445Cj+XjklCs5eeWY00hz5pwbunnnoKf5eeikTzl66ahrXm/qrEu+euuwm0527LT/+HrtuKOodu68e7h778AfiGHwxAsvYPHIy3df8sybx17z0G8XXvTUdw1c9djvVlv23MPmWffgP+1W+OSLn1P56CM9Vvrsq79a+/CTdl386AQCACH5BAUMAEEALBMAEwBTAlMCAAf/gEGCg4SFhoeIiYqLjI2Oj5CRkpOUlZaXkTmam5uYnp+goaKjpKWmp6ipqqusraucsLGarrS1tre4ubq7vL2+mbLBsb/ExcbHyMnKy8jCzrLM0dLT1NXW17bP2sPY3d7f4OHivtvlsOPo6err7O2C5vCc7vP09fb3rfH6s/j9/v8AAwbZR1CgwYMIE04jyFChw4cQI75iWFCixYsYMzKi2FCjx48gE3KkGLKkyZPuRnZEybKly4UqV76cSbNmrpg
```

Comment:
Missing close of tag.

Reply:
@jon-shipley the closing was there for the `img` it was just stack with the src so I added a space to make it clear.